lib/cli.ex

```diff
@@ -21,25 +21,32 @@ defmodule Onigumo.CLI do
             {working_dir, switches} = Keyword.pop(switches, :working_dir, File.cwd!())
 
             case switches do
-              [] -> module.main(working_dir)
-              _ -> usage_message()
+              [] ->
+                module.main(working_dir)
+
+              _ ->
+                OptionParser.to_argv(switches)
+                |> then(&"invalid OPTIONS #{Enum.join(&1, ", ")}")
```

Comment:
This can be DRIED to a private function accepting a list of invalid options in the form of argv.

```suggestion
invalid_options()
```